lib/next_ls.ex

```diff
@@ -124,7 +129,7 @@ defmodule NextLS do
          }
        },
        server_info: %{name: "Next LS"}
-     }, assign(lsp, root_uri: root_uri, workspace_folders: workspace_folders, client_capabilities: caps)}
+     }, assign(lsp, root_uri: root_uri, workspace_folders: workspace_folders, client_capabilities: caps, opts: opts)}
```

Comment:
before pushing it into the assigns, I think we should process them via a safe list and do the defaulting.